pkg/interfaces/contracts/pool-hooks/IECLPSurgeHook.sol

```diff
@@ -1,15 +0,0 @@
-// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
About this one, I'm extending it in a new PR to compute imbalance slopes (like, how fast the dynamic fee increases based on the imbalance), so I'd keep it. But I can undo this in the slope PR if needed.